### ISSUE TITLE
Mockup for photo and video attribution in reader slideshow

### ISSUE THREAD

**Issue by jesicarson** [participedia/frontend#895]
For photos and videos if any of the attribution fields are filled in, there will be a black, semi-transparent overlay that appears on the slideshow image, and the user entered text for 'attribution' will be a hyperlink if they entered a link. Title/description will follow. So there won't be static text/labels showing on view for these, only user entered text.

**Comment by jesicarson** [comment]
Example of case with youtube and vimeo videos added: https://participedia.xyz/case/5009. Need to figure out how an overlay for attribution would work for videos on desktop, without interfering with the player.  

![image](https://user-images.githubusercontent.com/13560530/44545612-6309ee80-a6ca-11e8-999b-c0efd3ccba0e.png)

**Comment by scottofletcher** [comment]
This looks great @jesicarson!  

One question/request I have: is it possible to have a pop-out slideshow if you click on the image (even if there's just one image)? There are some images that get cut off but the only way to see the whole thing is to save it and open it in w/a desktop app. 

I'm thinking something like Twitter when you click on attached media: 

**Regular Tweet:** 
![image](https://user-images.githubusercontent.com/29512491/44590826-eed65600-a770-11e8-9cee-955c7cd63f20.png)

**Click on Image:** 
![image](https://user-images.githubusercontent.com/29512491/44590843-f72e9100-a770-11e8-8744-270920c852d4.png)

**Mouse-over Image:** 
![image](https://user-images.githubusercontent.com/29512491/44590867-04e41680-a771-11e8-9fdb-0c43742e0ace.png)

**Comment by scottofletcher** [comment]
Oh and the same thing works for slideshows: 

**Regular Tweet**
![image](https://user-images.githubusercontent.com/29512491/44591050-89cf3000-a771-11e8-84f7-bdf1a8ea7070.png)

**Click on image:**
![image](https://user-images.githubusercontent.com/29512491/44591071-9784b580-a771-11e8-8787-23331649e60a.png)

**Slideshow:**
![image](https://user-images.githubusercontent.com/29512491/44591079-9b183c80-a771-11e8-94e7-5bb085648ce4.png)

**Comment by jesicarson** [comment]
Thanks @scottofletcher , full screen might be an option yes. I think the slideshow on desktop was meant to be that, essentially though. But that's not good that its cutting things off - can you send me an example of a case with a cut off image so I can share with dethe when we talk about this tomorrow? Thanks!

**Comment by scottofletcher** [comment]
@jesicarson Here's one with a cut off image: https://participedia.xyz/case/4947
If you save the image, you can see that's there's more to it (ie. the bottom 1/5 is cut off). There are loads more like this - any image that's not landscape gets the bottom lopped off. Most (basically all) pictures from Instagram are, therefore, a problem.

**Comment by jesicarson** [comment]
Yep definitely a problem with the slideshow format as it is. The only images that should be cropped are the lead image if it is needed to fit the homepage thumnail format. The rest of the images should show up un-cropped in the slideshow. I'm going to do a mockup for this anyway so I'll make sure to specify that. Thanks!
